Employee.h

```diff
@@ -0,0 +1,15 @@
+#import <Foundation/Foundation.h>
+#import "Organization.h" //forward declaration used here
+//@class Organization;
+@interface Employee : Person{
+    NSString *fullName;
+    Organization *organization;
+    NSString *employeeId;
+    
+}
+
+- (id)initWithName:(NSString *)fName andLastName:(NSString *)lName  andAge:(NSInteger)age andOrganization: (Organization *)orgName andEmpId:(NSString *)empId ;
```

Comment:
return type for init is preferred to be `instancetype` as it checks type of of instance returned at the compile time.